modules/ossm-rn-fixed-issues.adoc

```diff
@@ -19,6 +19,10 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {SMProductShortName} fixed issues
 
+* https://issues.redhat.com/browse/OSSM-3644[OSSM-3644] Federation egress-gateway gets wrong update of network gateway endpoints, causing extra endpoint entries. The federation egress-gateway has been updated so it gets the correct update of network gateway endpoints.
```

Comment:
We might change `The federation egress-gateway has been updated so it gets the correct update of network gateway endpoints.` to something like this `This has been fixed on the server side so it gets the correct update of network gateway endpoints`. WDYT ?

Reply:
Doc style prefers not to use phrases like "It is fixed" or "The issue is resolved" or "The error no longer presents" so how about ... `This has been updated on the server side so it gets the correct update of network gateway endpoints.`

Reply:
Per convo via Slack, updated to say `This has been updated on the server side so it gets the correct network gateway endpoints.`